server/pxf-hdfs/src/main/java/org/greenplum/pxf/plugins/hdfs/AvroFileAccessor.java

```diff
@@ -122,9 +133,21 @@ public OneRow readNextObject() throws IOException {
      */
     @Override
     public boolean openForWrite() throws Exception {
-        throw new UnsupportedOperationException();
+        // make writer
+        DatumWriter<GenericRecord> datumWriter = new GenericDatumWriter<>(schema);
+        writer = new DataFileWriter<>(datumWriter);
```

Comment:
```suggestion
        writer = new DataFileWriter<>(new GenericDatumWriter<GenericRecord>(schema));
```
to make it a 1-liner since you're not using `datumWriter` anywhere else.